redis/src/sentinel.rs

```diff
@@ -694,9 +694,7 @@ pub enum SentinelServerType {
     Replica,
 }
 
-/// LockedSentinelClient is a wrapper around SentinelClient since
```

Comment:
I think it's probably worth preserving the comment about the `&mut` ref (though just as a comment, not in the docs). It would be great to change that in the future :grin: